Help buttons modals

ObaltzerS opened the pull request:
created modals for all advanced search options other than the sort, bc no text was provided. It seems like there is more text, although I am not sure where to put it.

ObaltzerS commented:
I know you are working on the navbar, @halversm, have you created the options for "about us" "about this tool" and "contact us"?

ProfJanetDavis commented:
@ObaltzerS, can you comment on the Google Doc which content you weren't sure where to put?

ObaltzerS commented:
> @ObaltzerS, can you comment on the Google Doc which content you weren't sure where to put?

yeah, they are part of what @halversm was working on with the nabber.

ProfJanetDavis reviewed (changes requested):
A few things:
1. I like the looks of the `modalTemplate` function. But it looks like there are a bunch of unrelated changes in index.html. What are those about?
2. Is there a reason why you've added all the file content to the response object rather than fetching one static HTML file at a time into the modals?
3. Could you please merge my recent changes in the main branch? These are all related to deployment; there are no changes to index.html.

ProfJanetDavis commented:
It seems like you can click near the (?) icon but not actually on it. It's
super weird.

On Wed, Jul 19, 2023 at 4:01 PM Oliver Baltzer ***@***.***>
wrote:

> @ObaltzerS <https://github.com/ObaltzerS>, can you comment on the Google
> Doc which content you weren't sure where to put?
>
> yeah, they are part of what @halversm <https://github.com/halversm> was
> working on with the nabber.
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/PFAS-Project/governance-db-search-tool/pull/21#issuecomment-1642873544>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AEXJIU4ZLMV4QNZUZBQDYO3XRBROFANCNFSM6AAAAAA2QSJZP4>
> .
> You are receiving this because your review was requested.Message ID:
> ***@***.***>
>


-- 
Janet Davis (she/her)
Department Chair, Computer Science
Associate Professor and Microsoft Chair of Computer Science
Whitman College, Walla Walla, WA 99362
206-383-8798 (cell), http://cs.whitman.edu/~davisj/
Schedule an appointment: http://meetme.so/JanetDavis

ObaltzerS commented:
yeah, I think the unrelated changes you are seeing might have to do with me trying to format the help buttons, there was some weird bootstrap stuff going on and a <row> tag which was messing some things up. I added the filenames to the response object so that as I added files to the static folder, I wouldn't have to keep a list of each filename on the frontend. and yes, I'll see if I can merge the recent changes.

ObaltzerS commented:
I am aware of the button thing, I was hoping to ask you or Molly about it today because I was struggling with it but I forgot.

ObaltzerS commented:
also I don't know how to see exactly which changes you requested over GitHub. sorry

ObaltzerS commented:
looks like the missing spaces were aded when I merged

ProfJanetDavis commented:
Thanks for clarifying you just added the filenames to the response objects.
I think that's fine.

On Wed, Jul 19, 2023 at 4:32 PM Oliver Baltzer ***@***.***>
wrote:

> looks like the missing spaces were aded when I merged
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/PFAS-Project/governance-db-search-tool/pull/21#issuecomment-1642894773>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AEXJIUYDI5VUWCOBIYKBFX3XRBVCRANCNFSM6AAAAAA2QSJZP4>
> .
> You are receiving this because your review was requested.Message ID:
> ***@***.***>
>


-- 
Janet Davis (she/her)
Department Chair, Computer Science
Associate Professor and Microsoft Chair of Computer Science
Whitman College, Walla Walla, WA 99362
206-383-8798 (cell), http://cs.whitman.edu/~davisj/
Schedule an appointment: http://meetme.so/JanetDavis

ProfJanetDavis commented:
I see all the help button modals but not the modals for the "about" menu.
Are you still working on those?

On Wed, Jul 19, 2023 at 4:42 PM Janet Davis ***@***.***> wrote:

> Thanks for clarifying you just added the filenames to the response
> objects. I think that's fine.
>
> On Wed, Jul 19, 2023 at 4:32 PM Oliver Baltzer ***@***.***>
> wrote:
>
>> looks like the missing spaces were aded when I merged
>>
>> —
>> Reply to this email directly, view it on GitHub
>> <https://github.com/PFAS-Project/governance-db-search-tool/pull/21#issuecomment-1642894773>,
>> or unsubscribe
>> <https://github.com/notifications/unsubscribe-auth/AEXJIUYDI5VUWCOBIYKBFX3XRBVCRANCNFSM6AAAAAA2QSJZP4>
>> .
>> You are receiving this because your review was requested.Message ID:
>> ***@***.***>
>>
>
>
> --
> Janet Davis (she/her)
> Department Chair, Computer Science
> Associate Professor and Microsoft Chair of Computer Science
> Whitman College, Walla Walla, WA 99362
> 206-383-8798 (cell), http://cs.whitman.edu/~davisj/
> Schedule an appointment: http://meetme.so/JanetDavis
>


-- 
Janet Davis (she/her)
Department Chair, Computer Science
Associate Professor and Microsoft Chair of Computer Science
Whitman College, Walla Walla, WA 99362
206-383-8798 (cell), http://cs.whitman.edu/~davisj/
Schedule an appointment: http://meetme.so/JanetDavis

ObaltzerS commented:
yes
